internal/coreinternal/processor/filterconfig/config.go

```diff
@@ -24,32 +24,32 @@ import (
 // by the processor, captured under the 'include' and the second, exclude, to
 // define what is excluded from the processor.
 type MatchConfig struct {
-	// Include specifies the set of span/log properties that must be present in order
+	// Include specifies the set of span/log/metric properties that must be present in order
 	// for this processor to apply to it.
-	// Note: If `exclude` is specified, the span/log is compared against those
+	// Note: If `exclude` is specified, the span/log/metric is compared against those
```

Comment:
To avoid repeating `span/log/metric` everywhere it may be better to instead say `input data`, e.g.:
```suggestion
	// Note: If `exclude` is specified, input data is compared against those
```
Similarly in other places.

Reply:
This suggestion is perhaps a nit but it seems that it was dismissed (i.e. no change applied here or elsewhere) and the thread was resolved without a comment

Reply:
My apologies, you're correct, I'm going through and resolving when I change them locally vs. when I commit. I should probably add comments confirming that I'll do what's requested though.  Thanks for pointing that out!

Just to confirm I'll be changing the span/log/metric phrasing to match what Tigran's requested as "input data".

Reply:
It is best to resolve after the change is visible here in Github otherwise the reviewers have no way of confirming that it is done. It is also useful to comment on the issue saying what you did to resolve.